gulpfile.js

```diff
@@ -55,6 +55,7 @@ const {integration} = require('./build-system/tasks/integration');
 const {lint} = require('./build-system/tasks/lint');
 const {makeExtension} = require('./build-system/tasks/extension-generator');
 const {nailgunStart, nailgunStop} = require('./build-system/tasks/nailgun');
+const {performanceTest} = require('./build-system/tasks/performance-test');
```

Comment:
Since none of our other testing tasks (like `unit`, `integration`, `e2e`, or `visual-diff`) use the suffix `test` in the task name, I suggest the following changes:
- Rename `gulp performance-test` to `gulp performance`
- Rename `build-system/tasks/performance-test` to `build-system/tasks/performance`
- Rename `performance-test.js` to `performance.js`
- Rename `performanceTest()` to `performance()`